companion/src/mdichild.cpp

```diff
@@ -415,7 +415,8 @@ void MdiChild::generalEdit()
 
 void MdiChild::categoryAdd()
 {
-  CategoryData category("New category");
+  /*: Translators do NOT use accent for this, this is the default category name on Horus. */
+  CategoryData category(qPrintable(QObject::tr("Models")));
```

Comment:
Do not qualify with `QObject::` here.  You only need or want that when the code/class you're in doesn't already inherit from QObject.  When you qualify like this, Linguist will group the translations under a generic "QObject" heading (as you've seen before), which is not helpful, instead of with all the other translatable strings in ModelEdit.

`tr()` also accepts translator notes inline.  The better way to use `tr()` outside of QObjects is with [QCoreApplication::translate()](http://doc.qt.io/qt-5/qcoreapplication.html#translate)  which also accepts a "context" name (to group translations by subject/area) instead of putting everything in the default "QObject" translation group.